santosfamilyfoundation/RuinaMain
# [RUIN-199] Add some indication of which questions are required

# Description
Currently, there is no way to indicate which questions must be filled out. Now a red asterisk appears next to a question if it is required. With this update, there is nothing that prevents the user from submitting the form when required questions are not filled out. If this is something we want, leave a comment

# Testing
1. Start a new report
2. Check that a red asterisk appears next to each of the questions
